### DISCUSSION TITLE
四叶草支持？

### DISCUSSION THREAD

**Discussion by A2Sumie** [Tsuk1ko/CQ-picfinder-robot#442] [一般讨论]
手上的号用gocq都会code45了，只能被迫转四叶草了
如果出现可能四叶草相关的问题发issues会得到支持吗

**Comment by Tsuk1ko** [comment]
行的，不过四叶草也是兼容 onebot 的，理论上只会有四叶草自己的问题（

**Comment by Tsuk1ko** [comment]
有用 qsign 吗，我有个号也是不过用 qsign 还是能继续用 gocq

**Reply by A2Sumie** [reply]
2.1.0+老的用配置文件的qsign模拟8.9.63，没有试2.1.1，毕竟发布时qsign都被茶了（悲）。

**Comment by Tsuk1ko** [comment]
现在四叶草功能还是缺挺多的所以我也没有切过去，例如合并转发啥的

而且试了很多 x86 部署安卓虚拟机全以失败告终🥹我还没有废旧64位安卓机，难顶

**Reply by A2Sumie** [reply]
草，搜图真残了，要命（悲）

**Comment by Miuzarte** [comment]
还是看看远处的无头QQNT吧（，我的gocq现在估计退出就再也上不去了

**Reply by A2Sumie** [reply]
四叶草还是有维护的状态，感觉还行

**Reply by Tsuk1ko** [reply]
无头NT感觉也难搞（
